app/templates/components/pull-requests/card.hbs

```diff
@@ -0,0 +1,39 @@
+{{!-- TODO: Use @PR instead. --}}
+<li
+  class="pr-card"
+  data-test-pull-request
+>
+  <a
+    class="pr-title"
+    data-test-field="Title"
+    href={{PR.htmlUrl}}
+    rel="noopener noreferrer"
+    target="_blank"
+  >
+    {{PR.title}}
+  </a>
+
+  <span>
+    by
+    <a
+      href="https://github.com/{{PR.userLogin}}"
+      data-test-field="Author"
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      @{{PR.userLogin}}
+    </a>
+  </span>
+
+  <span>
+    in
+    <a
+      href={{PR.head.repo.html_url}}
```

Comment:
I don't think `PR.head.repo.html_url` and `PR.head.repo.name` work anymore. We'll later want to check the format in which GitHub (or perhaps `ember-data-github`) sends data.